R/indentation_linters.R

```diff
@@ -0,0 +1,46 @@
+#' @include utils.R
+#' @describeIn linters check that a certain kind of indentation is used
+#' @export
+indentation_linter <- function(spaces, linter_name = "indentation_linter") function(source_file) {
+  if (spaces %==% FALSE || spaces %==% 0L) {
+    allowed <- "\t"
+    message <- "Use tabs to indent."
+  } else {
+    nspc <- spaces  # else rex' internal 'spaces' is used!
+    allowed <- rex(n_times(" ", nspc))
+    message <- sprintf("Use %s spaces to indent.", digit_to_word(spaces))
```

Comment:
Rather than writing the word out I think we should just use the number directly.